crates/binstalk-downloader/Cargo.toml

```
@@ -91,6 +93,10 @@ cross-lang-fat-lto = ["zstd/fat-lto"]
 gh-api-client = ["json"]
 json = ["serde", "serde_json"]
 
+[target."cfg(windows)".dependencies]
+default-net = { version = "0.17.0", optional = true }
+ipconfig = { version = "0.3.2", optional = true }
```

We don't need the `ipconfig/computer` feature:

```suggestion
ipconfig = { version = "0.3.2", optional = true, default-features = false }
```